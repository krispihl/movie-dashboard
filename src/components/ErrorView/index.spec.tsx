import { ErrorView } from "./index";
import { render, screen } from '@testing-library/react';

describe('src/components/ErrorView', () => {
    it('should render Errorview correctly', () => {
        const { container } = render(<ErrorView />);

        expect(container).toMatchSnapshot();
    });

    it('should render Errorview correctly with full height', () => {
        const { container } = render(<ErrorView fullHeight />);

        expect(container).toMatchSnapshot();
        expect(screen.getByTestId('errorview')).toHaveClass('errorView--full');
    });
});
