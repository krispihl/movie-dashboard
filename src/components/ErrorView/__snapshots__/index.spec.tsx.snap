// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/ErrorView should render Errorview correctly 1`] = `
<div>
  <div
    class="errorView"
    data-testid="errorview"
  >
    <p
      class="errorView__notice"
    >
      Now this is unfortunate 
      <span
        class="material-icons"
      >
        sentiment_very_dissatisfied
      </span>
    </p>
    <p>
      Seems that something went wrong. Please try to reload.
    </p>
    <p>
      If the problem persists please reach out to our support popular[at]movies.com
    </p>
  </div>
</div>
`;

exports[`src/components/ErrorView should render Errorview correctly with full height 1`] = `
<div>
  <div
    class="errorView errorView--full"
    data-testid="errorview"
  >
    <p
      class="errorView__notice"
    >
      Now this is unfortunate 
      <span
        class="material-icons"
      >
        sentiment_very_dissatisfied
      </span>
    </p>
    <p>
      Seems that something went wrong. Please try to reload.
    </p>
    <p>
      If the problem persists please reach out to our support popular[at]movies.com
    </p>
  </div>
</div>
`;
